test/verify/check-apps

```diff
@@ -103,12 +102,21 @@ class TestApps(packagelib.PackageCase):
         b.wait_visible(".pf-v5-c-empty-state")
 
         self.createAppStreamPackage("app-1", "1.0", "1")
+        self.createAppStreamPackage("app-2", "1.0", "1", install=True)
         self.createAppStreamRepoPackage()
 
+        b.reload()
+        b.enter_page('/apps')
+
+        with b.wait_timeout(30):
+            b.wait_in_text(".pf-v5-c-alert.pf-m-inline h4.pf-v5-c-alert__title",
+                           "Warning alert:Appstream data package is missing, only installed applications are shown.")
```

Comment:
Please drop the "Warning alert:" prefix. It's not actual part of the string, but just some artifact of patternfly classes/CSS/whatever.